src/PageHeader/PageHeader.examples.stories.tsx

```diff
@@ -209,60 +282,61 @@ export const FilesPage = () => (
           </ActionMenu>
         </PageHeader.ContextAreaActions>
       </PageHeader.ContextArea>
-      <PageHeader.TitleArea>
+      {/* This is a hack for now. Breadcrumbs are not leading visuals but for now we are displaying them under until we get the layout right. */}
```

Comment:
@mperrotti How do you think we should handle the files page case where there is a breadcrumb component before the main title? I remember we briefly chatted about not to have the Breabcrumb under `PageHeader.LeadingAction` component which makes sense to me however, we haven't discussed it further. Would adding a special `PageHeader.Breadbrumb`  sub component be an option? I am curious what you think!

Reply:
Can you remind me why we can't continue to render it in `PageHeader.TitleArea`? That's what we do today. If we change the functionality of `PageHeader.TitleArea`, this could be a breaking change.

I think `PageHeader.Breadcrumb` is much better than rendering it in `PageHeader.LeadingAction`. We'll have to give this some more thought to make sure there's nothing else that should appear in the breadcrumb spot.